lib/widgets/footer_section.dart

```diff
@@ -0,0 +1,422 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:festival_flutterturkiye_org/utils/assets.dart';
+import 'package:festival_flutterturkiye_org/widgets/responsive_builder.dart';
+import 'package:material_design_icons_flutter/material_design_icons_flutter.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class FooterSection extends StatelessWidget {
+  const FooterSection({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ResponsiveBuilder(
+      largeWidget: const LargeWidget(),
+      mediumWidget: const MediumWidget(),
+      smallWidget: const SmallWidget(),
+    );
+  }
+}
+
+class SmallWidget extends StatelessWidget {
+  const SmallWidget({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
```

Comment:
Container already has an alignment. Let's remove this and add it into Container.